docs/serializer-formatter.rst

```diff
@@ -128,40 +129,6 @@ The Avro schema for the value will be registered under the subject *recordName-v
 *recordName* is the name of the Avro record. Because of this, the same Avro record type shouldn't
 be used in more than one topic.
 
-In the following example, we send a message with key of type ``string`` and value of type Avro record
```

Comment:
Yeah, I thought about adding it, but I also noticed that between

http://docs.confluent.io/2.0.1/app-development.html
http://docs.confluent.io/2.0.1/clients/index.html
http://docs.confluent.io/2.0.1/schema-registry/docs/serializer-formatter.html

We actually have quite a few docs + examples for clients, they are just scattered in sort of weird ways. We obviously would prefer the new consumer here and probably mention that the old consumer is heading towards deprecation, but I don't want to include that in this patch. We can do this in a follow up JIRA.